wallet-new/integration/Functions.hs

```diff
@@ -621,20 +617,6 @@ runAction wc action = do
 
         NoOp  -> pure ()
 
-    lastAction .= action
-
-    -- Let's print it out to JSON
-    walletState <- get
-
-    log "=================================================================="
-    log . decodeUtf8 . encodePretty $ diff (toJSON previousWalletState) (toJSON walletState)
-    log "=================================================================="
-
-    -- increment successful actions
-    log "Success!"
-    successActions <>= [action]
```

Comment:
Why is this block removed?

Reply:
Un-deleted it and the integration tests failed, so i removed it again and the integration tests still fail!!!

What's worse, these integration tests don't seem to be easily runnable locally locally.

Reply:
> What's worse, these integration tests don't seem to be easily runnable locally locally.
`nix-build release.nix -A walletIntegrationTests` seems to do the trick